src/index.css

```
@@ -1,50 +1,1423 @@
+  border-radius: 20px;
+  border: 1px solid var(--border-color);
+  background: var(--bg-card);
+  color: var(--text-primary);
+  cursor: pointer;
+  transition: transform 0.25s ease, box-shadow 0.2s ease;
+}
+
+.theme-toggle:hover {
+  transform: rotate(18deg) scale(1.1);
+  box-shadow: var(--shadow-card);
+=======
```

Stylelint (17.7.0)

[error] Unknown word ======= (CssSyntaxError)

(CssSyntaxError)